source/common/common/key_value_store_base.h

```diff
@@ -24,17 +27,20 @@ class KeyValueStoreBase : public KeyValueStore,
   // parses key value pairs from |contents| and inserts into store_.
   // Returns true on success and false on failure.
   bool parseContents(absl::string_view contents);
+  void ttlCallback(const std::vector<std::string>& keys);
```

Comment:
nit: Please comment. Instead of `ttlCallback()` how about something like `onExpiredKeys()`, or `onTtlExpired()`?